CHANGELOG.md

```diff
@@ -1,9 +1,24 @@
-# Change Log
+# Changelog
 
-All changes to this project will be documented in this file.
-This project adheres to [Semantic Versioning](http://semver.org/).
+All notable changes to this project will be documented in this file.
+Each new release typically also includes the latest modulesync defaults.
+These should not affect the functionality of the module.
 
-## 2017-01-12 - Release 1.1.0
+## [v2.0.0](https://github.com/voxpupuli/puppet-sftp_jail/tree/v2.0.0) (2017-10-12)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-sftp_jail/compare/v1.1.0...v2.0.0)
+
+**Closed issues:**
+
+- concat::fragment error when initing jail  [\#26](https://github.com/voxpupuli/puppet-sftp_jail/issues/26)
+
+**Merged pull requests:**
+
+- replace validate\_\* with datatypes in init [\#30](https://github.com/voxpupuli/puppet-sftp_jail/pull/30) ([bastelfreak](https://github.com/bastelfreak))
```

Comment:
Do you want to mark this as the breaking change?

Reply:
updated